apps/sms/js/thread_ui.js

```diff
@@ -2531,10 +2531,10 @@ var ThreadUI = {
       // eg. type "a", then delete it.
       // Always remove the the existing results.
       this.toggleRecipientSuggestions();
-      return;
+      return Promise.resolve(null);
```

Comment:
So I did it but I'm still not sure it's a good idea; to me, rejection should really be errors (as in "exception"), but here it's a totally normal and expected case. So i'd rather keep a resolved Promise after all.